com.ibm.wala.core.testdata/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -81,7 +81,7 @@ org.eclipse.jdt.core.compiler.problem.potentiallyUnclosedCloseable=error
 org.eclipse.jdt.core.compiler.problem.rawTypeReference=warning
 org.eclipse.jdt.core.compiler.problem.redundantNullAnnotation=error
 org.eclipse.jdt.core.compiler.problem.redundantNullCheck=error
-org.eclipse.jdt.core.compiler.problem.redundantSpecificationOfTypeArguments=warning
+org.eclipse.jdt.core.compiler.problem.redundantSpecificationOfTypeArguments=ignore
```

Comment:
The actual build of the testdata jar file is done with ant, via the `build.xml` file at the root directory.  But since this is testdata input anyway, I don't think we care about fixing these type of warnings.